### DISCUSSION TITLE
Question, how do we verify that a user has permission to list a classroom or venue as available?

### DISCUSSION THREAD

**Discussion by MiguelGibo** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#16] [Q&A]
While we could users self assign rooms, this approach might lead to many fake or joke listings. To avoid this, I’m looking for feedback on potential solutions or any other ideas the community might have.

- We could require documentation when trying to list a room, this would involve an admin on our end to review and approve the documents.
- We could have departmental or directorial accounts, accounts that belong to departments or directors who have the authority to assign rooms to other accounts.

What are your thoughts on these solutions?
Are there any other approaches we could consider verifying user permissions effectively?

**Comment by AngelUPR** [comment]
For now we will make it so that every user, student or organization, will be able to create events. Should this be an issue or a further topic of discussion we will create an issue
